assembly/assembly-ide-war/.classpath

```diff
@@ -6,24 +6,25 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources">
```

Comment:
I suggest to revert these changes for this PR as git fixup and rebase autosquash before merging. It's just to have clean commits.